src/Dapr.AspNetCore/DaprApplicationBuilderExtensions.cs

```diff
@@ -18,16 +19,17 @@ public static class DaprApplicationBuilderExtensions
         /// requests that use the cloud events structured format, allowing the event payload to be read directly.
         /// </summary>
         /// <param name="builder">An <see cref="IApplicationBuilder" />.</param>
+        /// <param name="jsonSerializerOptions">The <see cref="JsonSerializerOptions" /> to use when deserializing cloud event structured events.</param>
         /// <returns>The <see cref="IApplicationBuilder" />.</returns>
-        public static IApplicationBuilder UseCloudEvents(this IApplicationBuilder builder)
+        public static IApplicationBuilder UseCloudEvents(this IApplicationBuilder builder, JsonSerializerOptions jsonSerializerOptions = null)
```

Comment:
I don't think the cloud events middleware should have configurable settings. IIRC it only interacts with the *envelope* of the event, which is part of the specification.